src/UIView/UIView+Constraints.h

```diff
@@ -300,20 +310,20 @@
  *  @param view   the view to constrain
  *  @param offset the offset from centerY to bottom of view
  */
-- (void)constrainBottomOfView:(UIView *)view toCenterYWithOffset:(CGFloat)offset;
+- (NSLayoutConstraint *)constrainBottomOfView:(UIView *)view toCenterYWithOffset:(CGFloat)offset;
 
 /*!
  *  Constrain the view to 0 on all edges of caller
  *
  *  @param view the view to constrain
  */
-- (void)constrainViewToAllEdges:(UIView *)view;
+- (NSDictionary *)constrainViewToAllEdges:(UIView *)view;
```

Comment:
I think it'd be nice if these were supported for Swift. Add this to the top of the file:

```ObjC
typedef NSDictionary<NSString *, NSLayoutConstraint *> ConstraintDictionary;
```
Then use like this:

```
- (ConstraintDictionary *)constrainViewToAllEdges:(UIView *)view;
```